.env.old

```
@@ -0,0 +1,47 @@
+WEBUI_HOSTNAME=webui.${BASE_DOMAIN}
+PORTAL_HOSTNAME=portal.${BASE_DOMAIN}
+SEARXNG_HOSTNAME=searxng.${BASE_DOMAIN}
+SUPABASE_HOSTNAME=supabase.${BASE_DOMAIN}
+LANGFUSE_HOSTNAME=langfuse.${BASE_DOMAIN}
+OLLAMA_HOSTNAME=ollama.${BASE_DOMAIN}
+NEO4J_HOSTNAME=neo4j.${BASE_DOMAIN}
+TRAEFIK_DASHBOARD_HOST=traefik.${BASE_DOMAIN}
+
+# Postgres / Supabase
+POSTGRES_PASSWORD=Snk9Dz$F8t5xWPewW$H!*7T^jRUBvUlSs8VN%HNWhwiRkXb1
+JWT_SECRET=QY$sBW#J!^MSX-i1_Fe6fjAv5liVob1#rk1iOd1U*0ZCPnIU
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The JWT_SECRET key should go before the POSTGRES_PASSWORD key

(UnorderedKey)